surfactant/filetypeid/id_magic.py

```diff
@@ -208,6 +208,36 @@ def identify_file_type(filepath: str) -> Optional[str]:
                 if cm == 8:
                     if (cmf * 256 + flg) % 31 == 0:
                         return "ZLIB"
+            # cpio:
+            # https://www.systutorials.com/docs/linux/man/5-cpio/
+            if int.from_bytes(magic_bytes[:2], byteorder="big", signed=False) == 0o70707:
+                return "CPIO_BIN big"
+            if int.from_bytes(magic_bytes[:2], byteorder="little", signed=False) == 0o70707:
+                return "CPIO_BIN little"
+            if magic_bytes[:6] == b"070707":
+                return "CPIO_CHAR_OLD"
+            if magic_bytes[:6] == b"070701":
+                return "CPIO_ASCII_NEW"
```

Comment:
Should we also have a check for 070702? https://commons.apache.org/proper/commons-compress/apidocs/org/apache/commons/compress/archivers/cpio/CpioArchiveEntry.html